lib/rest_http.rb

```diff
@@ -0,0 +1,22 @@
+require 'json'
+require 'rest_client'
+
+class RestHttp
+  URL = "http://localhost:8083/chalyi".freeze
+
+  def get_request(endpoint, headers = {content_type: 'application/json'})
+    RestClient.get(URL + endpoint, headers)
+  end
+
+  def post_request(endpoint, payload, headers = {content_type: 'application/json'})
+    RestClient.post(URL + endpoint, payload, headers)
+  end
+
+  def put_request(endpoint, payload, headers = {content_type: 'application/json'})
+    RestClient.put(URL + endpoint, payload, headers)
+    end
```

Comment:
Seems like `end` key word is shifted because of 2 extra spaces (Or Github client shows it not correctly). If so then please check the file using Rubocop or Rubymine built-in static code analyzer.